ui/app/info.js

```diff
@@ -103,9 +103,9 @@ InfoScreen.prototype.render = function () {
             [
               h('div.fa.fa-support', [
                 h('a.info', {
-                  href: 'https://support.metamask.io',
+                  href: 'http://metamask.helpscoutdocs.com/',
```

Comment:
Https please

Reply:
Oh sorry, I thought ssl still wasn't fixed on this.